superflore/generate_installers.py

```diff
@@ -42,7 +43,7 @@ def generate_installers(
 
     info("Generating installers for distro '%s'" % distro_name)
     for i, pkg in enumerate(sorted(pkg_names[0])):
-        version = get_pkg_version(distro, pkg)
+        version = get_pkg_version(distro, pkg, kwargs.get('is_oe', False))
```

Comment:
@andre-rosa I'm not sure this change is in the spirit of this function. This is meant to be completely standalone. No checks should be made here. Maybe `kwargs` can be forwarded here to the `get_pkg_version` function and be evaluated there?

@tfoote do you have any suggestions for this? I'm not sure what the most pythonic way to do this is.

Reply:
I've forwarded ```kwargs``` to ```get_pkg_version``` and evaluated it there.